### PR TITLE
User reports relationship

### PR DESCRIPTION
**Pull request by kyle-rosenbluth**
Do we need tests for this?
@sandlerben

### CODE REVIEW

**Comment by sandlerben** [comment]
Please incorporate master so I can better tell what you added vs what was added recently in master

File: app/models/user.py [hunk 1]
```diff
@@ -62,6 +62,7 @@ class User(UserMixin, db.Model):
     password_hash = db.Column(db.String(128))
     role_id = db.Column(db.Integer, db.ForeignKey('roles.id'))
     agency_id = db.Column(db.Integer, db.ForeignKey('agencies.id'))
+    reported_incidents = db.Relationship("IdlingIncident")
```
Comment (sandlerben) [inline app/models/user.py]: Can we make the relationship in the IdlingIncident model just to be consistent with how we made other relationships to User?

Comment (sandlerben) [inline app/models/user.py]: Also, when creating a relationship make sure to pass arguments like `backref` and `lazy`!

**Comment by sandlerben** [comment]
Please test!

**Comment by kyle-rosenbluth** [comment]
Can we talk about how we would test something like this? Not quite sure what to do.

**Comment by sandlerben** [comment]
I would add one test to `test_user_model.py` and one test to `test_incident_report.py`. You can model them after a test case like `test_incident_report_with_agency_no_location`

**Comment by kyle-rosenbluth** [comment]
Ready for final review @sandlerben

**Comment by sandlerben** [comment]
:shipit:

**Pull request merged**